build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -104,7 +104,11 @@ service Execution {
   // send a [PreconditionFailure][google.rpc.PreconditionFailure] error detail
   // where, for each requested blob not present in the CAS, there is a
   // `Violation` with a `type` of `MISSING` and a `subject` of
-  // `"blobs/{hash}/{size}"` indicating the digest of the missing blob.
+  // `"blobs/{digest_function/}{hash}/{size}"` indicating the digest of the
```

Comment:
I think we should mention here the cases where `<digest_function>/` should be omitted and when it should be present.

Reply:
That's already mentioned here, right?
```
 // * `digest_function` is a lowercase string form of a `DigestFunction.Value`
 //   enum, indicating which digest function was used to compute `hash`. If the
 //   digest function used is one of MD5, MURMUR3, SHA1, SHA256, SHA384, SHA512,
 //   or VSO, this component MUST be omitted. In that case the server SHOULD
 //   infer the digest function using the length of the `hash` and the digest
 //   functions announced in the server's capabilities.
```

Reply:
Yes, but IMO it's easy to miss the reference if you're just reading the comments for this service- there's no direct link to the format. For that reason, I think it makes sense to mention it here as well.

Reply:
Done! Added one additional sentence:

> `digest_function` MUST thus omitted if its value is one of MD5, MURMUR3, SHA1, SHA256, SHA384, SHA512, or VSO.